container-action/scripts/configure_rit_locally.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+POWPEG_VERSION="${1:-SNAPSHOT-6.3.0.0}"
```

Comment:
Do we really want this script to be run if no parameter is given? Hardcoding it here means either having to keep it up-to-date to avoid running in the edge case were no parameter is given, having SNAPSHOT-6.3.0.0 being used and scratching our heads about the results.

I'd prefer to be explicit and fail if no parameter is given. You can implement that like

```suggestion
if [ $# -ne 1 ] || [ -z "${1:-}" ]; then
    echo "Usage: ${0##*/} POWPEG_VERSION" >&2
    exit 1
fi
POWPEG_VERSION=$1
shift
```